### PR TITLE
Added pending status and cancel url.

### PR DESCRIPTION
**Pull request by imeleshko**

### CODE REVIEW

File: upload/catalog/controller/extension/payment/coinbase.php [hunk 1]
```diff
@@ -157,7 +158,10 @@ public function updateRecord($data){
                 $status_message = 'Coinbase Commerce Status ' . $data['coinbaseStatus'] . ' Type ' . $data['type'];
 
                 if ($data['coinbaseStatus'] == 'NEW' && $data['type'] == 'charge:created') {
-                    $order_status = 'coinbase_created_status_id';  //Pending
+                    $order_status = 'coinbase_created_status_id';  //Created
+                } elseif ($data['coinbaseStatus'] == 'PENDING' && $data['type'] == 'charge:pending') {
+                    $order_status = 'payment_coinbase_pending_status_id';  //Pending
```
Comment (guacamoli) [inline upload/catalog/controller/extension/payment/coinbase.php:163]: does this handle cancelled?

Comment (imeleshko) [reply upload/catalog/controller/extension/payment/coinbase.php:163]: Yes 
upload/catalog/controller/extension/payment/coinbase.php

            "redirect_url" => $this->url->link('extension/payment/coinbase/redirect&orderId=' . $order_info["order_id"], true),
            "cancel_url" => $this->url->link('checkout/checkout')

Comment (guacamoli) [reply upload/catalog/controller/extension/payment/coinbase.php:163]: Cool, assuming this is still in your local branch?

Comment (imeleshko) [reply upload/catalog/controller/extension/payment/coinbase.php:163]: image
Current PR.

**Review by guacamoli — APPROVED** [review]

**Pull request merged**